addons/background/src/BackgroundPanel.js

```diff
@@ -118,7 +117,11 @@ BackgroundPanel.propTypes = {
     getQueryParam: PropTypes.func,
     setQueryParams: PropTypes.func,
   }).isRequired,
-  channel: PropTypes.instanceOf(EventEmitter),
+  channel: PropTypes.shape({
+    emit: PropTypes.func,
```

Comment:
shouldn't all of these required if the channel is supplied? should the channel be required as well?

Reply:
We have a defaultProp for it, so it shouldn't. As for particular shape, I basically just copypasted it from `addon/knobs`.

Reply:
Ideally, we should have this propTypes shape exported in `@storybook/addons` and used elsewhere. You can open a PR with such a refactoring if you feel like it =)